### DISCUSSION TITLE
SequentialFeatureSelector with scikit-learn Scalers

### DISCUSSION THREAD

**Discussion by ghost** [rasbt/mlxtend#902] [Q&A]
Hi

I'm using SequentialFeatureSelector with sklearn TimeSeriesSplit cross validation method.
Is there any way to use sklearn scalers with the pipeline above?

Thanks.

**Comment by rasbt** [comment]
Hm, I think you could just make a sklearn pipeline consisting of the scalers and the predictive model and provide that as inputs to the sequential feature selector.

**Reply by ghost** [reply]
Ok. I'm gonna try it.
Thank you.
